tests/live/test_workflow.py

```diff
@@ -54,6 +62,9 @@ def __call__(self, *args, load_output=True) -> dict:
             )
             pytest.fail(msg)
 
+        if proc.stderr:
```

Comment:
This makes it possible to see in `make test-live` what has been happening to the instance during creation.